engine/render/src/render/font.cpp

```diff
@@ -381,7 +381,7 @@ namespace dmRender
             ClearTexture(font_map, font_map->m_CacheWidth, font_map->m_CacheHeight);
 #endif
 
-        SetFontMapCacheSize(font_map, metrics->m_MaxWidth, metrics->m_MaxHeight, metrics->m_MaxAscent);
+        SetFontMapCacheSize(font_map, metrics->m_ImageMaxHeight, metrics->m_ImageMaxHeight, metrics->m_MaxAscent);
```

Comment:
The cell cache should use the bitmap image size

Reply:
Seems like it should be `metrics->m_ImageMaxWidth, metrics->m_ImageMaxHeight`. Right now both arguments are `height`